keyboards/handwired/co60/info.json

```
@@ -0,0 +1,38 @@
+{
+  "keyboard_name": "CO60",
+  "url": "https://github.com/jmdaly/CO60",
+  "maintainer": "qmk",
+  "width": 15,
+  "height": 5,
+  "layouts": {
+    "LAYOUT_all": {
+      "key_count": 68,
```

Line 10 has 67 keys designated, but you have 68 here (more on this issue below).

```suggestion
      "key_count": 67,
```